Documentation/building/unix-instructions.md

```diff
@@ -1,11 +1,9 @@
 Building CoreFX on FreeBSD, Linux and OS X
 ==========================================
 
-CoreFx can be built on top of current [Mono CI builds](#installing-mono-packages) or a direct [build/install of Mono](http://www.mono-project.com/docs/compiling-mono/). It builds using MSBuild and Roslyn and requires changes that have not yet made it to official released builds.
+Building CoreFX is pretty striaght-forward. Clone the repo and run the build script.
```

Comment:
Fixed thanks (although some do prefer straight-forward for uncomplicated stuff :smile:)